apps/iglidur-designer/src/app/libs/header/src/lib/toolbar/toolbar.component.ts

```diff
@@ -9,6 +9,7 @@ import { Subject, takeUntil } from 'rxjs';
 
 @Component({
   selector: 'lib-toolbar',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Why this?

Reply:
It's been resolved.. I added it while trying to fix an angular material custom theme issue.